### PR TITLE
Allow preserve structure in import

### LINKED ISSUE #21: File content should not be expanded during import
`kubectl konfig import --save ~/.kube/foo.config` seems expanding the content of file which is not expected.

```
# foo.config
users:
- name: vault-user
  user:
    client-certificate: client.crt
    client-key: client
```

is imported as

```
users:
- name: vault-user
  user:
    client-certificate: <content of client.crt>
    client-key: <content of client>
```

This is not expected since the certs generated by vault are temporary. It would be better to import it as is (the filename).

### PR DESCRIPTION
**Pull request by yujunz**
Fix #21

### CODE REVIEW

File: konfig [hunk 1]
```diff
@@ -86,20 +86,24 @@ merge() {
...
       error "unrecognized flag \"$1\""
     fi
     tmpcfg=$(mktemp konfig_XXXXXX)
     TMPFILES+=( "$tmpcfg" )
-    $KUBECTL config view --flatten > "$tmpcfg"
+    $KUBECTL config view --raw > "$tmpcfg"
 
     if [[ -z "$out" ]]; then
-      merge "$tmpcfg" "$@"
+      merge "$arg" "$tmpcfg" $@
```
Comment (corneliusweig) [inline konfig:104]:
```suggestion
      merge "$arg" "$tmpcfg" "$@"
```

File: konfig [hunk 2]
```diff
@@ -86,20 +86,24 @@ merge() {
...
     else
-      merge "$tmpcfg" "$@" > "$out"
+      merge "$arg" "$tmpcfg" $@ > "$out"
```
Comment (corneliusweig) [inline konfig:106]:
```suggestion
      merge "$arg" "$tmpcfg" "$@" > "$out"
```

**Review by corneliusweig — COMMENTED** [review]
Thank you for looking into this. It's great if users get the chance to choose if they want to preserve the structure or not. So this was definitely missing.

I have minimal remarks about the PR, but this already looks great. Nice work!

Please, could you also sign-off your commit? (see https://github.com/corneliusweig/konfig/blob/master/CONTRIBUTING.md#pull-request-checklist)

**Comment by yujunz** [comment]
> I have minimal remarks about the PR, but this already looks great. Nice work!

Thanks. All suggestions accepted, committed and squashed.

> Please, could you also sign-off your commit? (see https://github.com/corneliusweig/konfig/blob/master/CONTRIBUTING.md#pull-request-checklist)

Done.

**Comment by corneliusweig** [comment]
Thanks again for contributing!

**Pull request merged**